### DISCUSSION TITLE
Github Actions runs pages have stopped loading for swig project

### DISCUSSION THREAD

**Discussion by wsfulton** [community/community#13558] [Actions] [answered]
We are no longer able to view the details of Github Actions at https://github.com/swig/swig/actions. If you click on any of the recent builds, for example https://github.com/swig/swig/actions/runs/2042769109 we get the unicorn page and text:
```
This page is taking too long to load.

Sorry about that. Please try refreshing and contact us if the problem persists.
```
This has been happening for a over a week now and on different browsers and computers. It doesn't seem to happen with other projects on Github so seems peculiar to the swig project.

**Comment by ydshieh** [comment]
I saw other people reporting the same issue on community forum.
See https://github.community/t/workflow-receives-this-page-is-taking-too-long-to-load-message-with-unicorn/239583


It also happens to our project.

**Comment by ethomson** [comment, marked as answer]
Thanks for reporting this @wsfulton. We have identified a problem that is affecting a few users and causing unicorns. We should have a fix for this 🔜. Until then, I apologize for the inconvenience.

**Reply by ethomson** [reply]
We believe that we've deployed a fix for this issue. Please let us know if you're still having trouble here.

**Reply by wsfulton** [reply]
Fix confirmed, all working again now. Thanks.
